Changelog.rst

```diff
@@ -1,3 +1,12 @@
+version 3.3.5
+-------------
+----
+
+**xxxx-xx-xx**
+
+* Remove calls to ``LooseVersion`` from ``distutils`` due to deprecation of that package,
```

Comment:
As a general comment, realising that this message will likely change as we convert the approach used, ideally keep the line length shorter for the message in line with the approximate maximum for the rest of this file. Thanks.